integrations/tensorflow/WORKSPACE

```diff
@@ -7,7 +7,7 @@
 
 load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 
-TENSORFLOW_COMMIT = "5dd766f144ee0fc20506ee6476dc21c8e1816b69"
+TENSORFLOW_COMMIT = "b4767db1d3e8b95846d306bd0629f1f865fe76fd"
```

Comment:
There is a task on https://github.com/openxla/iree/issues/13037 to "Stop updating the tensorflow version as part of LLVM integrate."

Maybe revert this unless it's somehow still necessary?

Reply:
Let me drop that and see if it still builds...